docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  mysql:
+    image: mysql:5.7
+    restart: on-failure
+    environment:
+      - MYSQL_ROOT_PASSWORD=inventory_root_password
+      - MYSQL_HOST=inventory_mysql
+      - MYSQL_PORT=3306
+      - MYSQL_DATABASE=inventory_local
+      - MYSQL_USER=inventory_test_user
+      - MYSQL_PASSWORD=inventory_test_password
+    entrypoint: ['docker-entrypoint.sh', '--default-authentication-plugin=mysql_native_password']
+    ports:
+      - "4444:3306"
+    volumes:
+      - ./.data/mysql:/var/lib/mysql:delegated
+    container_name: inventory_mysql
```

Comment:
a better name would be `course_inventory_mysql`?

Reply:
Done.